state-chain/runtime/src/migrations/solana_integration.rs

```diff
@@ -86,68 +86,107 @@ impl OnRuntimeUpgrade for SolanaIntegration {
 			},
 		));
 
-		let (vault_address, genesis_hash, usdc_address): (SolAddress, Option<SolHash>, SolAddress) =
-			match cf_runtime_upgrade_utilities::genesis_hashes::genesis_hash::<Runtime>() {
-				cf_runtime_upgrade_utilities::genesis_hashes::BERGHAIN => {
-					log::warn!("Need to set up Solana integration for Berghain");
-					(
+		let (vault_address, genesis_hash, usdc_address, durable_nonces_and_accounts): (
+			SolAddress,
+			Option<SolHash>,
+			SolAddress,
+			Vec<DurableNonceAndAccount>,
+		) = match cf_runtime_upgrade_utilities::genesis_hashes::genesis_hash::<Runtime>() {
+			cf_runtime_upgrade_utilities::genesis_hashes::BERGHAIN => {
+				log::warn!("Need to set up Solana integration for Berghain");
```

Comment:
log::error! , we don't want to miss this.

Reply:
i dont understand, this is not an error. its just an info log. if anything, we should make it log::info!

Reply:
We have missed this kind of thing in the past. It's easy to skim over info and warnings. An error shows up clearly. It doesn't prevent the migration checks from passing.